Conflict between StateVariable and ResonantFilter enum filter types

In file included from C:\Users\jibin\Documents\Arduino\helios4_6\helios4_6.ino:108:0:
C:\Users\jibin\Documents\Arduino\libraries\Mozzi-master/StateVariable.h:53:6: error: multiple definition of 'enum filter_types'
 enum filter_types { LOWPASS, BANDPASS, HIGHPASS, NOTCH };
      ^~~~~~~~~~~~
In file included from C:\Users\jibin\Documents\Arduino\libraries\Mozzi-master/LowPassFilter.h:16:0,
                 from C:\Users\jibin\Documents\Arduino\helios4_6\helios4_6.ino:100:
C:\Users\jibin\Documents\Arduino\libraries\Mozzi-master/ResonantFilter.h:71:6: note: previous definition here
 enum filter_types { LOWPASS, BANDPASS, HIGHPASS, NOTCH };
      ^~~~~~~~~~~~

exit status 1

Compilation error: exit status 1

--- a/StateVariable.h
+++ b/StateVariable.h
@@ -49,8 +49,9 @@ Soc., Vol 45, No. 9, 1997 September
 #include "meta.h"
 #include "mozzi_fixmath.h"
 #include "mozzi_utils.h"
+#include "ResonantFilter.h"
 
-enum filter_types { LOWPASS, BANDPASS, HIGHPASS, NOTCH };
+//enum filter_types { LOWPASS, BANDPASS, HIGHPASS, NOTCH };
 
 /** A State Variable filter which offers 12db resonant low, high, bandpass and
 notch modes.

